data/T/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Threshold Network Token",
+  "symbol": "T",
+  "decimals": 18,
+  "description": "Threshold Network is the home of $tBTC, proxy re-encryption #PRE, $thUSD, and all threshold cryptography.",
+  "website": "https://threshold.network",
+  "twitter": "@TheTNetwork",
+  "tokens": {
+    "ethereum": {
+      "address": "0xCdF7028ceAB81fA0C6971208e83fa7872994beE5"
+    },
+    "optimism": {
+      "address": "0x747e42Eb0591547a0ab429B3627816208c734EA7"
```

Comment:
This contract doesn't seem to be verified on optimism etherscan yet. Can you verify this contract?

Reply:
Hi @jakim929, thanks for looking here.

I can't seem to verify the contract, and receive this: "Error! Unable to generate Contract ByteCode and ABI" 

Do you think you can take a look please? 

It seems that the Optimism Bridge 3 is behaving differently after the fork, and creating additional internal transactions to what it did before. The byte code for our contract deployment is different to those prior to the fork. Could this be the reason the verification fails?

Here is the L2 ERC20 token contract creation: https://optimistic.etherscan.io/tx/0xad69e07c124ada374d23131333c9d892d64559488fbc85df44751adf76b66981

Reply:
Contract is now verified.

https://optimistic.etherscan.io/address/0x747e42eb0591547a0ab429b3627816208c734ea7

Reply:
thank you!